extra/1-radio-stations.js

```diff
@@ -30,8 +30,29 @@
  * ======= TESTS - DO NOT MODIFY =======
  * Note: You are not expected to understand everything below this comment!
  */
+ function getAllFrequencies(){
+   let frequencyA = [];
+  for (let i = 87; i < 109; i++) {
+    frequencyA.push(i);
+  }
+  return frequencyA;
+}
+
+
+function getStations(){
+  let radioStation = [];
+  let  allFrequencies = getAllFrequencies();
+ for (let i = 0; i < allFrequencies.length - 1; i++) {
+    if (isRadioStation(allFrequencies[i])) {
+    radioStation.push(allFrequencies[i]);
+    }
+  }
+    return radioStation;
+}
```

Comment:
Well done on working on an extra question 👍 ⭐ The more challenge you solve, the better you'll learn